CHANGELOG.unreleased.md

```diff
@@ -32,5 +33,8 @@
 
 - vhd-lib major
 - xo-vmdk-to-vhd patch
+- @xen-orchestra/backups
+- @xen-orchestra/backups-cli
+- @xen-orchestra/proxy
```

Comment:
- missing version qualification
- `backups-cli` does not do any backup, no need to release it

```suggestion
- @xen-orchestra/backups patch
- @xen-orchestra/proxy patch
```